app/index.js

```diff
@@ -126,6 +126,10 @@ module.exports = yeoman.Base.extend({
     },
 
     gdtBase: function() {
+      // Save the existing composer.json file if any. See composerJson().
+      if (this.fs.exists('composer.json')) {
+        this.fs.copy('composer.json', 'composer.orig');
```

Comment:
Yep, knew it was virtual therefore not a big performance issue.  But yeah, reading in the actual json values is probably better.  Will update.